### PR TITLE
feat: verify hostname (load free certificate call to bunny cdn)

### PR DESCRIPTION
**Pull request by EmperorOrokuSaki**

### CODE REVIEW

File: serverless/src/functions/apps/handler.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    const bunnyCdn = new BunnyCdn(process.env.BUNNY_CDN_ACCESS_KEY);
+    const hostname = JSON.parse(event.body).hostname;
+
+    let args: LoadFreeCertificateMethodArgs = {
+      hostname,
+    };
+
+    await bunnyCdn.loadFreeCertificate(args).catch((e) => {
+      throw e;
+    });
```
Comment (zoruka) [inline serverless/src/functions/apps/handler.ts:29]: This code is the same than:
```suggestion
    await bunnyCdn.loadFreeCertificate(args);
```

File: serverless/src/libs/bunnyCDN.ts [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+  public name: string;
+
+  constructor({ name, message }: BunnyCdnErrorOptions) {
+    super(message);
+
+    this.name = name;
+    this.message = message;
+  }
+}
+const BUNNY_CDN_API_URL = 'https://api.bunny.net';
```
Comment (zoruka) [inline serverless/src/libs/bunnyCDN.ts:18]: should this be from `env`?

**Review by zoruka — APPROVED** [review]
this PR comes with changes related to #264, maybe check if need to merge both

File: serverless/src/libs/bunnyCDN.ts (hunk 2 above)
Comment (EmperorOrokuSaki) [reply serverless/src/libs/bunnyCDN.ts:18]: It's not going to change.

**Pull request merged**